src/index.html

```diff
@@ -9,6 +9,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="nav_link">
+        <img src="images/logo.png"
+          alt="MOYO-logo"
+        >
+      </a>
```

Comment:
This link do not belong nav block
```suggestion
      <a href="#home" class="logo">
        <img src="images/logo.png"
          alt="MOYO-logo"
        >
      </a>
```